content/influxdb/cloud/organizations/buckets/bucket-schema/_index.md

```diff
@@ -41,37 +41,67 @@ Use the `influx` CLI to set the schema-type and measurement schemas for your buc
     ```
 
 2. Use your text editor to create a schema columns file for each measurement you want to add.
-The file defines the column names, tags, fields, and data types to require for a measurement.
-Format the file as CSV, JSON, or [Newline delimited JSON (NDJSON)](http://ndjson.org/),
-as in the following examples:
-{{% code-tabs-wrapper %}}
-{{% code-tabs %}}
-[usage-resources.csv](#)
-[usage-cpu.json](#)
-[sensor.ndjson](#)
-{{% /code-tabs %}}
-{{% code-tab-content %}}
-```sh
-{{< get-shared-text "bucket-schema/usage-resources.csv" >}}
-```
-{{% /code-tab-content %}}
-{{% code-tab-content %}}
-```json
-{{< get-shared-text "bucket-schema/usage-cpu.json" >}}
-```
-{{% /code-tab-content %}}
-{{% code-tab-content %}}
-```json
-{{< get-shared-text "bucket-schema/sensor.ndjson" >}}
-```
-{{% /code-tab-content %}}
-{{% /code-tabs-wrapper %}}
+   The file defines the column names, tags, fields, and data types to require for a measurement.
+
+   Format the file as CSV, JSON, or [Newline delimited JSON (NDJSON)](http://ndjson.org/),
+   as in the following examples:
+   {{% code-tabs-wrapper %}}
+   {{% code-tabs %}}
+   [usage-resources.csv](#)
+   [usage-cpu.json](#)
+   [sensor.ndjson](#)
+   {{% /code-tabs %}}
+   {{% code-tab-content %}}
+   ```sh
+   {{< get-shared-text "bucket-schema/usage-resources.csv" >}}
+   ```
+   {{% /code-tab-content %}}
+   {{% code-tab-content %}}
+   ```json
+   {{< get-shared-text "bucket-schema/usage-cpu.json" >}}
+   ```
+   {{% /code-tab-content %}}
+   {{% code-tab-content %}}
+   ```json
+   {{< get-shared-text "bucket-schema/sensor.ndjson" >}}
+   ```
+   {{% /code-tab-content %}}
+   {{% /code-tabs-wrapper %}}
+
+   #### Write valid schemas 
+   To ensure your schema is valid, review [InfluxDB data elements](/influxdb/cloud/reference/key-concepts/data-elements/) and [naming restrictions](/influxdb/cloud/reference/syntax/line-protocol/#naming-restrictions).
```

Comment:
Being the only naming restriction is don't start w underscore (and we say that in line 78, seems like we can drop from line 72 to simplify this intro statement and so folks don't click a link they don't need to?

Reply:
```suggestion
   To ensure your schema is valid, review [InfluxDB data elements](/influxdb/cloud/reference/key-concepts/data-elements/).
```